com.unity.multiplayer.mlapi/Runtime/Transports/UNET/UNetChannel.cs

```diff
@@ -12,11 +16,37 @@ public class UNetChannel
         /// <summary>
         /// The name of the channel
         /// </summary>
-        public NetworkChannel Id;
+        #if UNITY_EDITOR
+        [ReadOnly]
+        #endif
+        public byte Id;
```

Comment:
I'd defer to @mattwalsh-unity for this change. I believe he converted things from `string` to `byte` to `enum`, he should be the judge here.

Reply:
It's the only way though. This is just the UNET transport iterpretation, not the core lib. If we don't support byte here we can't support the UI.

Reply:
oh, I see!